Server/server/controllers/ServerController.ts

```diff
@@ -0,0 +1,45 @@
+import WebSocket from 'ws';
+import Player from './Player';
+import WaitingRoom from './WaitingRoom';
+import Player from '../models/Player';
```

Comment:
These imports look weird, they should reference classes held in `/models`.

Reply:
Imports should now be fixed